app/models/notification.rb

```diff
@@ -24,6 +25,11 @@ class Notification < ApplicationRecord
   scope :unread,   ->(unread)       { where(unread: unread) }
   scope :owner,    ->(owner_name)   { where(repository_owner_name: owner_name) }
 
+  scope :state,    ->(state) { joins(:subject).where('subjects.state = ?', state) }
+
+  scope :subjectable, -> { where(subject_type: ['Issue', 'PullRequest', 'Commit', 'Release']) }
```

Comment:
Seems weird for some reason. Aren't all notifications 'subjectable' anyway? Or maybe it's more appropriate to say that subjectable means the presence of a `subject_url` 🤔